Parsers/ASimAuthentication/Tests/VMwareCarbonBackCloud_ASimAuthentication_DataTest.csv

```diff
@@ -0,0 +1,8 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 199 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Carbon Black Cloud""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 199 records (100.0%) for field [EventVendor] of type [Enumerated]: [""VMware""] (Schema:Authentication)"
+"(2) Info: Empty value in 191 records (95.98%)  in recommended field [EventResultDetails] (Schema:Authentication)"
+"(2) Info: Empty value in 196 records (98.49%)  in optional field [EventOriginalResultDetails] (Schema:Authentication)"
+"(2) Info: Empty value in 3 records (1.51%)  in recommended field [DvcIpAddr] (Schema:Authentication)"
+"(2) Info: Empty value in 3 records (1.51%)  in recommended field [SrcIpAddr] (Schema:Authentication)"
```

Comment:
Why SrcIpAddr is coming null in some cases?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/aac37a24-17ee-426f-b7c5-5dc06e769de9)

Reply:
We have few logs where clientIp_s field is empty which is mapped with SrcIpAddr.